src/gp/src/GPMSA.C

```diff
@@ -766,7 +767,8 @@ GPMSAFactory<V, M>::setUpEmulator()
 
   for (unsigned int i=0; i != m_numSimulations; ++i)
     for (unsigned int j=0; j != numOutputs; ++j)
-      (*simulationOutputMeans)[j] += (*m_simulationOutputs[i])[j];
+      (*simulationOutputMeans)[j] +=
```

Comment:
Verify that this isn't doubly applying the mean shift between the code here and in normalized_output.